### DISCUSSION TITLE
[PLUGIN SUGGESTION] Admin LTE integration

### DISCUSSION THREAD

**Discussion by kengketa** [TiagoSilvaPereira/vemto-issues#137] [Ideas & Suggestions]
**Is your feature request related to a problem? Please describe.**
Admin LTE integration

**Describe the solution you'd like**
Apart of Tailwind I think Admin LTE is also the most popular and flexible to adapt with any projects

**Comment by TiagoSilvaPereira** [comment]
@kengketa thank you for your suggestion... I think this type of feature will best fit as a Vemto plugin. I want to add plugin creation capabilities to Vemto in the next months. Cheers

**Comment by ddiaz2380** [comment]
Can I add it as a theme? It would be great to be able to have in vemto adminlte, Coreui. Greetings

**Comment by TiagoSilvaPereira** [comment]
@ddiaz2380 thank you for your suggestion... I'll add plugins capabilities in the future, so you would be able to create your own themes and integrations
